src/site/content/en/learn/css/lists/index.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Lists
+description: >
+  A list, structurally, is composed of a list container element, like an `<ol>` or `<ul>`, and filled with list items, (`<li>`). In this module, you'll learn how to style all the parts of a list.
+audio:
+  title: 'The CSS Podcast - 030: Lists'
+  src: 'https://traffic.libsyn.com/secure/thecsspodcast/TCP030_v2.mp3?dest-id=1891556'
+  thumbnail: image/foR0vJZKULb5AGJExlazy1xYDgI2/ECDb0qa4TB7yUsHwBic8.png
+authors:
+  - emmatwersky
+date: 2021-08-30
+---
+
+Imagine you have a bunch of items you plan to buy during your next grocery trip. One common way to represent this visually is a list—but how can you add styling to your grocery list?
+
+```html
+<ul>
+  <li>oat milk</li>
+  <li>rhubarb</li>
+  <li>cereal</li>
+  <li>pie crust</li>
+</ul>
+```
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'zYzrEOW',
+  tab: 'result'
+} %}
+
+## Creating a List
+
+The preceding list started with a semantic element, or `<ul>`, with children groceries list items, or `<li>`'s. If you inspect each `<li>` element you can see that they all have `display: list-item`, which is why the browser renders it with a `::marker` by default.
+
+```css
+li {
+  display: list-item;
+}
+```
+
+There are two other types of lists. 
+
+Ordered lists can be created with `<ol>`, in which case the list-items `::marker` be a number by default. 
+
+```html
+<ol>
+  <li>oat milk</li>
+  <li>rhubarb</li>
+  <li>cereal</li>
+  <li>pie crust</li>
+</ol>
+```
+
+And description lists are created with `<dl>`, however this list type does not use the `<li>` list item element.
+
+```html
+<dl>
+  <dt>oat milk</dt>
+  <dd>- non dairy trendy drink</dd>
+  <dt>cereal</dt>
+  <dd>- breakfast food</dd>
+</dl>
+```
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'WNOrZNG',
+  tab: 'result'
+} %}
+
+## List Styles
+
+Now that you know how to make a list, you can style them. The first CSS properties to discover are those that are applied to the entire list.
+
+There are three list-style properties you can use to style your example: `list-style-position`, `list-style-image`, and `list-style-type`.
+
+### `list-style-position`
+
+[`list-style-position`](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-position) allows you to move your bullet point to either `inside` or `outside` the list-item's contents. The default `outside` means the bullet point is not included in the list items contents while `inside` moves the first element among the list item's contents.
+
+{% Img src="image/dQwiPAoChjTYzXYCNg6U6Tsnr3x1/qVlpVCrwKwbGjEJdmqh0.jpg", alt="A list with both outside and inside ::marker which shows that outside (default value) is not in the list-item and inside is inside the list-item content box", width="800", height="270" %}
+
+{% Codepen {
```

Comment:
this would be clearer if the list items had a background color.

Reply:
added!